src/test/java/net/snowflake/client/jdbc/ResultJsonParserV2Test.java

```diff
@@ -223,4 +229,41 @@ public static String stringToHex(String input) {
     }
     return sb.toString().toUpperCase();
   }
+
+  @Test
+  public void testObjectMapperWithLargeJsonString() throws Exception {
```

Comment:
let's make this test parameterized with different sizes as parameters - you can also add bigger sizes that you tested up to 128MB

Reply:
Updated test cases to test 16MB all the way up to 128MB.

Reply:
let's use the `@RunWith(Parameterized.class)` instead of for loop